web/app.fluidity.money/app/queries/useUserActionsAggregate.ts

```diff
@@ -10,19 +10,21 @@ export type AggregatedTransaction = Omit<
   swap_in: boolean;
   type: "send" | "swap";
   timestamp: string;
+  rewardTier: number;
```

Comment:
i don't know if i like having this be permanently modified to include lootbottle info. if we plan to revert all this later i'd say it's fine, but if this is staying imo the transaction types should be explicitly `AggregatedTransaction | AggregatedLootBottleTransaction`, and have different behaviour for when the airdrop is/isn't running. otherwise `aggregated_user_transactions` will end up the same as the problem it was trying to solve (left joins tacked on in a series of migrations)